devito/operators.py

```diff
@@ -0,0 +1,40 @@
+from jit_manager import JitManager
+from propagator import Propagator
+import os
+
+
+class Operator(object):
```

Comment:
This file has been renamed in master to `operator.py`. I don't think this re-introduction is intended.